README.md

```diff
@@ -23,3 +23,34 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
+
+| left       | operator | right    | result     | note              |
+| ---------- | -------- | -------- | ---------- | ----------------- |
+| Time       | -        | Time     | Duration   |                   |
+| Time       | +=       | Time     | Time       | seems unexpected? |
+| Time       | +        | Duration | Time       |                   |
+| Time       | +=       | Duration | Time       |                   |
+| Time       | -        | Duration | Time       |                   |
+| Time       | -=       | Duration | Time       |                   |
+| TimeWindow | +        | Duration | TimeWindow |                   |
```

Comment:
It's not clear if it changes the TimeWindow start, end, or both. In such case I guess it's better to use methods with clear names instead of operators.

Reply:
Good point, I removed them in favor of a method.